package/itest/ubuntu.sh

```diff
@@ -34,9 +34,20 @@ apt-get install -y --force-yes python3.7 python3-pip python3-yaml awscli
 dpkg -i /dist/${DISTRIB_CODENAME}/clusterman_${PACKAGE_VERSION}_amd64.deb || true
 apt-get install -y --force-yes --fix-broken
 
+# Sometimes our acceptance tests run in parallel on the same box, so we need to use different CIDR ranges
+if [ "${DISTRIB_CODENAME}" = "xenial" ]; then
+    CIDR_BLOCK="10.0.0.0/24"
+else
+    CIDR_BLOCK="11.0.0.0/24"
```

Comment:
Are you sure about this? IIRC `11.0.0.0` isn't considered a private subnet

Reply:
True, I also copied this block from `service-itest-runner`. I'll change it to at least use proper networks